css/index.css

```diff
@@ -1,3 +1,432 @@
+
+/* .main-content1 .features {
+
+}
+
+.main-content1 .about {
+	
+} */
+
+.middle-image img{
+	margin: 20px auto;
+	width: 100%;
+}
+
+.main-content2 {
+	display: flex;
+	width: 100%;
+	margin: 10px 0;
+    height: 200px;
+}
+
+.main-content2 .services {
+	text-align: left;
+	display: flex;
+	justify-content: flex-start;
+	flex-direction: column;
+	padding: 0;
+}
+
+.main-content2 .product {
+	text-align: left;
+	display: flex;
+	flex-direction: column;
+	padding: 0;
+}
+
+.main-content2 .vision {
+	text-align: left;
+	display: flex;
+	flex-direction: column;
+	padding: 0;
+}
+
+.main-content2 h2 {
+	margin-bottom: 20px;
+	display: flex;
+	flex-direction: row;
+	justify-content: flex-start;
+}
+
+.main-content2 p {
+	display: flex;
+	margin: 2px;
+	width: 100%;
+}
+
+.contact {
+	margin: 20px 0;
+}
+
+.footer {
+	margin: 20px;
+	padding: 20px;
+	text-align: center;
+}
+
+
+/* Services Page  */
+
+.servicespage .top-image {
+	margin-top: 37px;
+	width: 100%;
+}
+
+.servicespage .services {
+	height: 100px;
+	width: 900px;
+	margin-top: 15px;
+	display: flex;
+	flex-wrap: wrap;
+}
+
+.servicespage .box {
+	width: 435px;
+	height: 200px;
+	border: 1px solid black;
+	margin-top: 30px;
+	padding: 10px;
+	background: whitesmoke;
+}
+
+.servicespage .main-content {
+	display: flex;
+	flex-wrap: wrap;
+	justify-content: space-between;
+	margin-bottom: 40px;
+}
+
+.servicespage .main-content .learn-more {
+	background: white; 
+	margin: 25px 0;
+	width: 150px;
+	border-radius: 10%;
+	box-shadow: remove;
+	border-style: solid;
+	border: 1px solid black;
+	height: 35px;
+}
+.servicespage .bottom-section {
+	display: flex;
+	margin-top: 40px;
+}
+
+.servicespage .some-facts {
+	margin: 20px 0;
+	width: 450px;
+	margin-right: 40px;
+}
+
+.servicespage .some-facts li {
+	list-style: disc;
+	margin: 10px 20px;
+}
+
+.servicespage footer {
+	text-align: center;
+	margin: 50px;
+}
+
+@media (max-width: 800px) {
+    .main-header img {
+        display: none;
+    }
+
+    .container {
+        width: 95%;
+    }
+
+    .nav {
+        flex-direction: column;
+    }
+
+    .nav .logo-container {
+        order: -1;
+        display: flex;
+        justify-content: center;
+    }
+
+    .main-header .big-text {
+        width: 100%;
+        justify-content: center;
+    } 
+
+    .main-header .big-text .get-started-container p{
+        width:;
+        justify-content: center;
+        margin-left: 50px;
+
+    }
+
+}
+
+@media (max-width: 500px) {
+
+    .container {
+        width: 101%;
+    }
+
+    .mobile-container {
+        width: 95%;
+        margin: 0 auto;
+    }
+
+    .nav-no-logo{
+        flex-direction: column;
+        display: flex;
+    }
+
+    .logo-container {
+        height: 25px;
+    }
+
+    .nav {
+        display: flex; 
+        flex-direction: column;
+        justify-content: space-between;
+    }
+    
+    .nav a {
+        padding: 16px 0;
+        width: 100%;
+        text-align: center;
+        border: 1px solid gray;
```

Comment:
You can fix the double border touch with border-bottom and select only the first child to have border-top.